[32mINFO    [0m root:test_calculator.py:87 Starting test for valid subtraction operation with login.
[32mINFO    [0m root:test_calculator.py:94 Attempting to log in with credentials: {'userName': 'admin', 'password': '123'}
[32mINFO    [0m root:test_calculator.py:96 Login response status code: 200
[32mINFO    [0m root:test_calculator.py:99 Login response data: {'message': None, 'data': {'success': True, 'token': 'ec299663-7617-4a5b-bdbf-140c928a3392'}, 'status': 'success'}
[32mINFO    [0m root:test_calculator.py:101 Login response data: {'message': None, 'data': {'success': True, 'token': 'ec299663-7617-4a5b-bdbf-140c928a3392'}, 'status': 'success'}
[32mINFO    [0m root:test_calculator.py:103 Received token: ec299663-7617-4a5b-bdbf-140c928a3392
[32mINFO    [0m root:test_calculator.py:111 Sending calculation request: {'op1': 10, 'operation': '-', 'op2': 3}
[32mINFO    [0m root:test_calculator.py:116 Received response with status code: 200
[32mINFO    [0m root:test_calculator.py:119 Calculation response data: {'message': None, 'data': 7, 'status': 'success'}
[32mINFO    [0m root:test_calculator.py:125 The result of the addition operation is correct.